mathphys/arbeitsweise.tex

```diff
@@ -23,7 +23,17 @@ \subsection{Arbeitskreise und Arbeitsgruppen}
 Damit nicht die ganze Fachschaftssitzung darüber diskutieren muss, ob man neben Brettspielen auch Rollenspiele anbietet, gibt es verschiedene Arbeitskreise (AKs) der Fachschaft, die bei Bedarf in den Sitzungen berichten. Wenn eine Entscheidung oder Geld benötigt wird, entscheidet die Sitzung. Dort werden auch AK-Gründungen bekanntgegeben.
 
 \vspace*{-2mm}
+\subsection{AK-Queer}
+In der Hochschulgruppe PR$\sqrt{-1}$DE treffen sich queere mathphysinfo Studis zum quatschen, vernetzen und gemeinsamen Feierabend feiern.
 
+Wir freuen uns über neue Gesichter im neuen Semester, komm doch einfach mal vorbei! Absprachen und Ankündigungen finden sich in der über den QR Code zugänglichen WhatsApp-Gruppe.
+
+Bei Fragen oder Ähnlichem kannst Du dich sonst auch an ak-queer@mathphys.info wenden.
+\begin{figure}[h]
+    \includegraphics*[width=\linewidth]{ak_queer_qr.png}
```

Comment:
The link in the qrcode is broken:
![grafik](https://user-images.githubusercontent.com/26827864/188980082-1681d81c-c993-46f4-a1ac-9fe109d056ba.png)

Why dont you just use QRCode Package:
```latex
....
\usepackage{qrcode}
...
\qrcode[height=8em, link]{https://mathphys.info/\~chris/studieninformationstag.pdf}
```

Reply:
QR - Code was given by the workshop. We do not know the concrete link in order to generate the QR-Code.